src/Orleans/Messaging/Message.cs

```diff
@@ -81,10 +83,12 @@ public static class Metadata
         private ActivationAddress targetAddress;
         private ActivationAddress sendingAddress;
         private static readonly Logger logger;
-        
+        private static readonly IObjectPool<Message> pool;
```

Comment:
globalPool?  To clearly distinguish it from the Pool property in PooledResource..?